src/postgraphile/http/createPostGraphileHttpRequestHandler.ts

```diff
@@ -387,7 +387,9 @@ export default function createPostGraphileHttpRequestHandler(
           /<\/head>/,
           `  <script>window.POSTGRAPHILE_CONFIG=${safeJSONStringify({
             graphqlUrl: `${externalUrlBase}${graphqlRoute}`,
-            streamUrl: watchPg ? `${externalUrlBase}${graphqlRoute}/stream` : null,
+            streamUrl: watchPg
+              ? `${externalUrlBase}${graphqlRoute}`.replace(/\/*$/, '') + `/stream`
```

Comment:
I’m not at my computer; but would this be equivalent? If so, I’d much prefer it for DRY, simplicity and performance reasons 👍

```suggestion
              ? `${externalUrlBase}${streamRoute}`
```

Reply:
yes, I didn't notice before that `externalUrlBase` can't end with à `/`